docker-compose.yml

```diff
@@ -96,21 +95,19 @@ services:
     environment:
       - AUTOGRAPH_URL=http://autograph-app-hsm:8001/
       - AUTOGRAPH_KEY=19zd4w3xirb5syjgdx8atq6g91m03bdsmzjifs2oddivswlu9qs
+      - AUTOGRAPH_ROOT_HASH
     links:
       - app-hsm
     depends_on:
       - app-hsm
     volumes:
       - hsmtmpdir:/tmp/
-      - hsmconfig:/hsmapp/
```

Comment:
Unused apart from `/app/gen-pki.out`, which the monitor doesn't need.

It was also masking the hash cacert script.